crates/fluvio-stream-dispatcher/src/metadata/local.rs

```diff
@@ -256,460 +91,630 @@ impl LocalMetadataItem {
     }
 }
 
-#[derive(Debug)]
-struct SpecStore {
-    version: AtomicU64,
-    data: RwLock<HashMap<String, SpecPointer>>,
-    sender: Sender<SpecUpdate>,
-    receiver: Receiver<SpecUpdate>,
-    path: PathBuf,
-}
+cfg_if::cfg_if! {
```

Comment:
The only change here is that removed the feature flag k8 for the whole metadata mod and put this if for the things that really need this flag. Then it's possible to import `LocalMetadataItem` without the k8 flag.

I can move it to another PR if it's not very clear.

Reply:
Yes, it would be better made as separate PR

Reply:
Done with: https://github.com/infinyon/fluvio/pull/4092